packages/core/src/app/ui/form/ChecklistItem.tsx

```diff
@@ -14,6 +14,7 @@ export interface ChecklistItemProps {
     htmlId?: string;
     isDisabled?: boolean;
     label: ReactNode | ((isSelected: boolean) => ReactNode);
+    description?: ReactNode | (() => ReactNode);
```

Comment:
Is there any chance to pass a string to description?

Reply:
Yes. We have this opportunity thanks to ReactNode